server/src/internalClusterTest/java/org/elasticsearch/action/search/CCSPointInTimeIT.java

```diff
@@ -71,6 +105,112 @@ public void testBasic() {
                 .get();
             assertNoFailures(resp);
             assertHitCount(resp, (includeLocalIndex ? localNumDocs : 0) + remoteNumDocs);
+
+            SearchResponse.Clusters clusters = resp.getClusters();
+            int expectedNumClusters = 1 + (includeLocalIndex ? 1 : 0);
+            assertThat(clusters.getTotal(), equalTo(expectedNumClusters));
+            assertThat(clusters.getSuccessful(), equalTo(expectedNumClusters));
+            assertThat(clusters.getSkipped(), equalTo(0));
+
+            if (includeLocalIndex) {
+                AtomicReference<SearchResponse.Cluster> localClusterRef = clusters.getCluster(RemoteClusterAware.LOCAL_CLUSTER_GROUP_KEY);
+                assertNotNull(localClusterRef);
```

Comment:
NIT: Maybe extract something like:

```java
   private static void assertOneSuccessfulShard(AtomicReference<SearchResponse.Cluster> clusterRef) {
        assertNotNull(clusterRef);
        SearchResponse.Cluster cluster = clusterRef.get();
        assertThat(cluster.getTotalShards(), equalTo(1));
        assertThat(cluster.getSuccessfulShards(), equalTo(1));
        assertThat(cluster.getFailedShards(), equalTo(0));
        assertThat(cluster.getFailures().size(), equalTo(0));
        assertThat(cluster.getStatus(), equalTo(SearchResponse.Cluster.Status.SUCCESSFUL));
        assertThat(cluster.getTook().millis(), greaterThanOrEqualTo(0L));
        assertFalse(cluster.isTimedOut());
    }
```

to dry up the two duplicate blocks of assertions here?